### ISSUE TITLE
Cheat Finder

### ISSUE THREAD

**Issue by stochaztic** [SourMesen/Mesen-S#88]
As mentioned by others in #48, I'm looking forward to seeing a Cheat Finder in Mesen-S. I'm aware of your comments in that issue that you plan to work on a new UI in the future. I'd like to request that, if it's not too much work, the existing Mesen UI is brought over to tide users over until that can occur. Mesen-S is a fantastic tool in all other aspects for my dev and debugging workflow, and I'd love to be able to condense all work on this emulator and not need to open any others regularly. This is really the only missing step keeping this from being achievable.

If after consideration you decide to continue to wait, I would of course understand, but an open issue for Cheat Finder that I can track progress of to know when the feature is implemented would be welcome. Thanks for your time and efforts!

**Comment by podstanar** [comment]
Yes and also adding "_Step Back_" and "_Edit Selected Code/Edit Subroutine_" debugger features from **Mesen** would make life easier for game-hacking related stuff. Use them all the time when making NES codes.
